options/locale/locale_en-US.ini

```
@@ -589,12 +589,12 @@ passcode_invalid = The passcode is incorrect. Try again.
 twofa_enrolled = Your account has been enrolled into two-factor authentication. Store your scratch token (%s) in a safe place as it is only shown once!
 
 u2f_desc = Security keys are hardware devices containing cryptographic keys. They can be used for two-factor authentication. Security keys must support the <a rel="noreferrer" href="https://fidoalliance.org/">FIDO U2F</a> standard.
-u2f_require_twofa = Your account must be enrolled in two-factor authentication to use security keys.
 u2f_register_key = Add Security Key
 u2f_nickname = Nickname
 u2f_press_button = Press the button on your security key to register it.
 u2f_delete_key = Remove Security Key
 u2f_delete_key_desc = If you remove a security key you can no longer sign in with it. Continue?
+u2f_warning_no_totp = "Warning: You have no TOTP devices enrolled. Consider adding one as a backup in case your U2F token is lost."
```

Suggesting to use TOTP as backup is rather poor security because TOTP is inherently lower security and attack vector.

```suggestion
u2f_warning_no_totp = "Warning: You have no TOTP devices enrolled. Consider adding one in order to access your account on devices that do not support U2F."
```

---

I agree with you completely, but given the feedback from the community so far, I didn't think the PR would ever be merged without a warning like that.